src/app/styl/Official_-_Experimental_theme.styl

```diff
@@ -39,8 +39,8 @@ $Pure_White = #FFFFFF
     $FullScreenButtonHover = $Pure_White
 
     $FilterBarBg = $NavColor
-    $FilterBarText = #c3c5c7
```

Comment:
then there is no point in having a hover color if you set the same for both. UX is better, imo, with changes when you hover. Especially in this case where filterbar is not important and the attention should be dragged below, to the list